test/basic.html

```diff
@@ -51,6 +51,10 @@
             assert.equal(pages.selected, undefined);
           });
 
+          test('null activateEvent', function() {
```

Comment:
Not sure I follow. Are you saying the test should indicate why `activateEvent` should be null? Why isn't this equally arbitrary to testing that `nothing is selected` or `selectedClass == 'iron-selected'`. It looks to me like testing for "`property` equals `value`" is the pattern here.